Cloud-CV/EvalAI
# #2291 Github OAuth Integration

> What is Added
- OAuth (Github) using Django Allauth & Django RestAuth 

>Testing Steps
- Create an OAuth APP on Github, Redirect URL should be `<frontend-url>/auth/oauth-redirect/github/`.
- Set the FrontEnd URL in the environment variable FRONTEND_URL.
- Change client id in frontend (login page)  
- Follow Django AllAuth Documentation to add a site
